docker/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+node bin/sourcecred.js load $@
+yarn start
```

Comment:
As per #1288 I would not suggest running webpack dev-server as root as the default entrypoint for a public facing docker image. Should you be going for a production build, perhaps building static files into an nginx container or to be committed for gh-pages is a lot more secure.